h2o-test-support/src/main/java/water/TestUtil.java

```diff
@@ -390,6 +390,13 @@ public static Frame parse_test_file( String fname) {
   public static Frame parse_test_file( String fname, int[] skipped_columns) {
     return parse_test_file(Key.make(), fname, skipped_columns);
   }
+  
+  /** Find & parse & track in {@link Scope} a CSV file.  NPE if file not found.
+   *  @param fname Test filename
+   *  @return      Frame or NPE */
+  public static Frame parse_and_track_test_file(String fname) {
+    return Scope.track(parse_test_file(Key.make(), fname));
```

Comment:
What is the difference between this new method and the `public static Frame parse_test_file( String fname) ` (line 390)? Maybe some copying error?

Reply:
It does the same job _plus_ it puts the frame to the scope.

Reply:
Aaa, I see. I did not noticed the Scope part. :facepalm: